src/main/scala/uk/gov/hmrc/playcrosscompilation/AbstractPlayCrossCompilation.scala

```diff
@@ -63,7 +68,8 @@ abstract class AbstractPlayCrossCompilation(
     },
     unmanagedResourceDirectories in Test += {
       (sourceDirectory in Test).value / playDir / "resources"
-    }
+    },
+    crossScalaVersions ~= playCorssScalaBuilds
```

Comment:
how does it work when teams already define `crossScalaVersions` in their build.sbt, which version wins?

Reply:
depends on order in `settings` they are applied. In most cases `playCrossCompilationSettings` are added as latest.
This would just scan previously defined list and in case of play-25 would left only "2.11" and "2.10" versions

Reply:
ok, that makes sense